Sources/PrimerSDK/Classes/Extensions & Utilities/StringExtension.swift

```diff
@@ -43,10 +43,86 @@ internal extension String {
         let inputP = NSPredicate(format: "SELF MATCHES %@", regex)
         return inputP.evaluate(with: self)
     }
+    
+    var isOnlyLatinCharacters: Bool {
+        let set = CharacterSet(charactersIn: "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLKMNOPQRSTUVWXYZ")
```

Comment:
Is Göran Åkerman a valid cardholder name? I though only the english alphabet letters are allowed.

Reply:
Good point, yeah I think it's only ASCII characters. However, the thing is, what would the user expect? And does the card issuer reject the payment if Göran Åkerman puts `Göran Åkerman` instead of `Goran Akerman`? I think we should align across the SDKs on this one. Let's discuss.